### PR TITLE
Fixed grammar and spelling

### PR DESCRIPTION
**Pull request by ALifeLivedFully**
Lines 22, 26, 76, and 225.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ pandas >= 1.2.0
 numpy >= 1.20.0
 scipy >= 1.6.0
 ```
-The solution - manually install these package before installing alpaca-trade-api. e.g:
+The solution - manually install these package's before installing alpaca-trade-api. e.g:
```
Comment (haxdds) [inline README.md:22]:
```suggestion
The solution - manually install these packages before installing alpaca-trade-api. e.g:
```

Comment (haxdds) [reply README.md:22]: should be plural instead of possessive

**Review by haxdds — CHANGES_REQUESTED** [review]
Thanks for contributing! @ALifeLivedFully 

Just one small comment

File: README.md (hunk 1 above)
Comment (ALifeLivedFully) [reply README.md:22]: Good catch!

**Review by haxdds — APPROVED** [review]
Thank you

**Pull request merged**